### ISSUE TITLE
npm cheack fail

### ISSUE THREAD

**Issue by deadwarrior61** [leon-ai/leon#181]
- Leon version: 1.0.0-beta.2
- OS (or browser) version:last debian
- Node.js version:10.15.2
- Complete "npm run check" output:
➡ /bin/sh -c node --version
✔ v10.15.2

➡ /bin/sh -c npm --version
✔ 5.8.0

➡ /bin/sh -c pipenv --version
✔ pipenv, version 11.9.0

✖ Error: Command failed: /bin/sh -c pipenv run python --version
Creating a virtualenv for this project…
Using /usr/bin/python3.7m (3.7.3) to create virtualenv…
Warning: it looks like virtualenv is not in your PATH. We cannot continue until this is resolved.

please how can i fix this, my python default version is 3.7 and in pipfile i change python version to 3.7 but it don't work anymore. leon run but when say something, he say :

Sorry, the Greeting module of my Leon package is broken!

Sorry, I've got an error with the Greeting module of my Leon package!

**Comment by g-w1** [comment]
I am having the same problem
```➜  leon git:(master) ✗ npm run check 
> leon@1.0.0-beta.2 check /home/jacob/repos/leon 
> babel-node scripts/run-check.js 
⠋ 
--- 
.: CHECKING :. 
➡ /bin/sh -c node --version 
✔ v10.19.0 
➡ /bin/sh -c npm --version 
✔ 5.8.0 
➡ /bin/sh -c pipenv --version 
✔ pipenv, version 2018.11.26 
✖ Error: Command failed: /bin/sh -c pipenv run python --version 
Warning: Python 3.6 was not found on your system… 
You can specify specific versions of Python with: 
  $ pipenv --python path/to/python 
➜  leon git:(master) ✗ npm run check 
> leon@1.0.0-beta.2 check /home/jacob/repos/leon 
> babel-node scripts/run-check.js 
⠋ 
--- 
.: CHECKING :. 
➡ /bin/sh -c node --version 
✔ v10.19.0 
➡ /bin/sh -c npm --version 
✔ 5.8.0 
➡ /bin/sh -c pipenv --version 
✔ pipenv, version 2018.11.26 
✖ Error: Command failed: /bin/sh -c pipenv run python --version 
Warning: Python 3.6 was not found on your system… 
You can specify specific versions of Python with: 
  $ pipenv --python path/to/python 
```

**Comment by Djaymare** [comment]
> * Leon version: 1.0.0-beta.2
> * OS (or browser) version:last debian
> * Node.js version:10.15.2
> * Complete "npm run check" output:
>   ➡ /bin/sh -c node --version
>   ✔ v10.15.2
> 
> ➡ /bin/sh -c npm --version
> ✔ 5.8.0
> 
> ➡ /bin/sh -c pipenv --version
> ✔ pipenv, version 11.9.0
> 
> ✖ Error: Command failed: /bin/sh -c pipenv run python --version
> Creating a virtualenv for this project…
> Using /usr/bin/python3.7m (3.7.3) to create virtualenv…
> Warning: it looks like virtualenv is not in your PATH. We cannot continue until this is resolved.
> 
> please how can i fix this, my python default version is 3.7 and in pipfile i change python version to 3.7 but it don't work anymore. leon run but when say something, he say :
> 
> Sorry, the Greeting module of my Leon package is broken!
> 
> Sorry, I've got an error with the Greeting module of my Leon package!

Try : 
# sudo apt-get install python-virtualenv



> Leon: v 1.0.0-beta.2
> Node: 10.19.0
> Python 3.7.3
> os: debian 10
> I am having the same problem
> 
> ```
> > leon@1.0.0-beta.2 check /home/jacob/repos/leon 
> > babel-node scripts/run-check.js 
> ⠋ 
> --- 
> .: CHECKING :. 
> ➡ /bin/sh -c node --version 
> ✔ v10.19.0 
> ➡ /bin/sh -c npm --version 
> ✔ 5.8.0 
> ➡ /bin/sh -c pipenv --version 
> ✔ pipenv, version 2018.11.26 
> ✖ Error: Command failed: /bin/sh -c pipenv run python --version 
> Warning: Python 3.6 was not found on your system… 
> You can specify specific versions of Python with: 
>   $ pipenv --python path/to/python 
> ➜  leon git:(master) ✗ npm run check 
> > leon@1.0.0-beta.2 check /home/jacob/repos/leon 
> > babel-node scripts/run-check.js 
> ⠋ 
> --- 
> .: CHECKING :. 
> ➡ /bin/sh -c node --version 
> ✔ v10.19.0 
> ➡ /bin/sh -c npm --version 
> ✔ 5.8.0 
> ➡ /bin/sh -c pipenv --version 
> ✔ pipenv, version 2018.11.26 
> ✖ Error: Command failed: /bin/sh -c pipenv run python --version 
> Warning: Python 3.6 was not found on your system… 
> You can specify specific versions of Python with: 
>   $ pipenv --python path/to/python 
> ```

It isn't the same issue : https://github.com/leon-ai/leon/issues/138#issuecomment-605828443

**Comment by Divlo** [comment]
Thanks for your report!
Could you please try with the latest version of the `develop` branch ?
Also there is now Gitpod, so you can easily start Leon directly in your browser, it is worth taking a look: https://gitpod.io/#https://github.com/leon-ai/leon
